backend/auth/passport.js

```diff
@@ -0,0 +1,15 @@
+const helper = require('./helpers/helper');
+
+const jwtCallback = async (jwt_payload, done) => {
+  const user = await helper.getUserByEmail(jwt_payload.email);
+
+  if (user) {
+    return done(null, user);
+  }
+
+  return done(null, false);
```

Comment:
return done(null, user) and return done(null, false)
false is not opposit to user (object)